contents/uix/design-systems.md

```diff
@@ -0,0 +1,130 @@
+
+## What is a Design System
+
+The following definition best encapsulates the essence of a Design System.
+> A Design System is the Single Source of Truth which groups all the elements that will allow the teams to design, realize and develop a product. <sub>--[Blog post from UX Collective](https://uxdesign.cc/everything-you-need-to-know-about-design-systems-54b109851969)</sub>
+
+The typical structure of a Design System looks like this:
+<center>
+<img src="https://cdn-images-1.medium.com/max/2600/1*wSHUJh069L618oXqVZYDtA.png" alt="MySQL Workbench GUI Software" width="95%">
+
+_Figure 1. Typical structure of a Design System_ ([source](https://uxdesign.cc/can-design-systems-fix-the-relationship-between-designers-developers-eb12fc9329ab))
+</center>
+
+Below is a brief explanation of the parts that make up a Design System. The explanation includes reference to a living example, [Polaris](https://polaris.shopify.com/), Shopify's Design System.
+- `Rules`: Rules include abstract elements such as brand values, mindset, and shared beliefs. For example, one of Polaris' [product experience principles](https://polaris.shopify.com/patterns-and-guides/product-experience-principles#navigation) is to `Put merchants first`. The principle promotes thinking about the needs of different types of merchants and be intentional about how to respond to them in the product or feature.
+- `Pattern Library`: A Pattern Library integrates functional components and their usage. For example, Shopify's [product component library](https://github.com/Shopify/polaris-react) contains a custom [Datepicker](https://github.com/Shopify/polaris-react/tree/master/src/components/DatePicker) component for merchants to select a date range.
+- `Building Blocks`: Building Blocks refer to documentation consisting of the collection of interface elements. Polaris has a section which describes their [components](https://polaris.shopify.com/components/get-started) in detail. For example, the custom [Datepicker](https://polaris.shopify.com/components/forms/date-picker#navigation) component is clearly documented with different examples.
+- `Style Guide`: A Style Guide focuses on graphic styles such as colors, fonts, illustrations, etc and their usage. For example, Polaris' [style guide](https://polaris.shopify.com/design/colors#section-color-usage) recommends to use Indigo for buttons and avoid using Indigo for text links.
+
+## Why should we use a Design System?
+
+<center>
+<img src="https://cdn-images-1.medium.com/max/2600/1*wFOwLNi-xGBfyGnssynQaQ.png" alt="Typical Design and Development process" width="95%">
+
+_Figure 2. Typical Design and Development process_ ([source](https://uxdesign.cc/can-design-systems-fix-the-relationship-between-designers-developers-eb12fc9329ab))
+</center>
+
+Consider the typical design and development process. Designers come up with sketches, wireframes and hand them over to the developers to implement them through code. Here's the problem:
```

Comment:
>and hands

Reply:
I'm using `Designers` here, so `hand` should be correct.